.github/workflows/release.yml

```diff
@@ -20,6 +23,10 @@ jobs:
         python-version: "3.12"
     - name: Setup uv
       uses: astral-sh/setup-uv@v3
+    - name: Set up Deno
+      uses: denoland/setup-deno@v2
+    - name: Install Deno Jupyter Kernel
+      run: deno jupyter --install
```

Comment:
issue (ci perf): I don't think we need this to build the package and create a GitHub release (nor the env var right above).